addons/mail/static/src/components/rtc_video/rtc_video.js

```diff
@@ -60,11 +60,13 @@ export class RtcVideo extends Component {
     //--------------------------------------------------------------------------
 
     /**
+     * Plays the video as some browsers may not support or block autoplay.
+     *
      * @private
      * @param {Event} ev
      */
     async _onVideoLoadedMetaData(ev) {
-        await ev.target.play();
+        await ev.target.play().catch(()=>{});
```

Comment:
lint: should have space around `=>`

Ideally: should only catch the kind of exception that you expect and throw again the others.

But at least add a comment explaining why there is exception and why you are ok to ignore it.